src/packages/engine-core/src/NAPIEngine.ts

```diff
@@ -355,7 +354,6 @@ You may have to run ${chalk.greenBright(
     if (this.libraryStarted) {
       // eslint-disable-next-line no-async-promise-executor
       this.libraryStoppingPromise = new Promise(async (res) => {
-        await this.runBeforeStop()
```

Comment:
Uh where did this go instead? Where are we even triggering this then?

Reply:
It was removed, inorder to mirror the functionality of the `NodeEngine`